src/components/RemediationDetailsTable.js

```diff
@@ -45,6 +45,20 @@ function resolutionDescriptionCell (remediation, issue) {
     );
 }
 
+function issueDescriptionCell (issue) {
+    const url = buildIssueUrl(issue.id);
+
+    if (url) {
+        return (
+            <span>
```

Comment:
Can we remove the span tags here and instead change the `cells` in the buildRows to use this as a title for this cell?

```
{
                isOpen: false,
                selected: this.state.selected[issue.id] || false,
                cells: [
                    {
                        title: issueDescriptionCell(issue)
                    },
                    resolutionDescriptionCell(remediation, issue),
                    issue.resolution.needs_reboot === true ? 'Yes' : 'No',
                    issue.systems.length,
                    {
                        title: getIssueApplication(issue),
                        props: { className: 'ins-m-nowrap' }
                    }
                ]
            },
.
.
.
```